miqi/agent/tools/apply_patch.py

```
@@ -327,6 +349,32 @@ async def execute(self, **kwargs: Any) -> str:  # type: ignore[override]
+                    base_dir=_base,
+                    workspace_root=self._base_workspace or self._workspace,
+                    shared=shared,
+                    granted=_granted,
+                    once_granted=_once,
+                    write_resolver=self._write_resolver,
+                    persist_extra_root=self._persist_extra_root,
+                    boundary_enforced=boundary_enforced,
+                    bypass=self._bypass_approval,
+                )
+                if _auth is None:
+                    return f"Error: 权限被拒绝：用户未授权写入 {_p}"
```

Ruff (0.16.2)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)